### ISSUE TITLE
XO Experiments views tracking for DXA.Java does not work

### ISSUE THREAD

**Issue by NiclasCedermalm** [sdl/dxa-modules#28]
The XO module for DXA.Java does not track experiment views, only conversions.
The class XOTrackingMarkupDecorator needs to be extended with a call to the analytics manager to track views each time a specific experiment variant is shown:

```
@Override
protected String processMarkupByAnalytics(HtmlNode markup, ViewModel model) {
    SmartTargetExperiment experiment = (SmartTargetExperiment) model.getMvcData().getMetadata().get("Promotion");
    try {
        **this.analyticsManager.trackView(experiment.getExperimentDimensions(), Collections.emptyMap());**
        return this.analyticsManager.addTrackingToLinks(markup.toHtml(), experiment.getExperimentDimensions(), Collections.emptyMap());
    } catch (SmartTargetException e) {
        log.warn("Exception while adding tracking to experiment links", e);
        return markup.toHtml();
    }
}
```

**Comment by azarakovskiy** [comment]
TSI-2996 internally

**Comment by rpannekoek** [comment]
Fixed in DXA 2.0
